web/docs/auth/auth-hooks.md

```diff
@@ -520,6 +528,54 @@ export const onAfterLogin: OnAfterLoginHook = async ({
 
 Read more about the data the `onAfterLogin` hook receives in the [API Reference](#the-onafterlogin-hook).
 
+### Refreshing the OAuth access token
+
+If you are using OAuth, you can use the `onAfterSignup` or `onAfterLogin` hooks to refresh the OAuth access token if the provider supports it.
+
+You'll need to import the OAuth client from the `wasp/server/oauth` module for the provider you are using. The OAuth client has a `refreshAccessToken` method that you can use to refresh the access token.
+
+Here is an example of how you can refresh the Google OAuth access token:
```

Comment:
```suggestion
Here's an example of how you can refresh the access token for Google OAuth:
```
Subjective, but since we're talking about "refreshing the access token" all the time, I recommend keeping that phrase together, and appending the specific provider at the end.